### DISCUSSION TITLE
Is there any plan to deploy to personal servers to access our app over the network？

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1320] [Ideas / Feature Requests]
It's like turn on-air=True ,but from a personal server

**Comment by rodja** [comment]
Do you mean deployment of your app to a remote server? Or using a relay server which will allow the access of edge devices through the cloud?

**Reply by v479038280** [reply]
I don't know much about the web, I wonder if I can deploy an app locally, and if this computer is connected to the Internet, can other connected computers access this app in any way，not only in the local area network

**Reply by rodja** [reply]
In this case On Air is exactly what you are looking for. Right now it is only possible to get a one-time URL with 1 hour validity. But we are hard working on creating a login service which will allow you to set a token on your local app to make it accessible world wide through a well defined url.

**Comment by falkoschindler** [comment]
I'll close this feature request, since it's already possible with NiceGUI On Air.
It is currently only a tech preview, but will be announced more prominently and improved continuously over the next months and years.
